Makefile

```diff
@@ -22,6 +22,7 @@ test-coverage:
 			rm profile.out ; \
 		fi \
 	done
+	awk 'BEGIN{print "mode: atomic"} /^(.+):([0-9]+).([0-9]+),([0-9]+).([0-9]+) ([0-9]+) ([0-9]+)$$/ {print}' coverage.txt > coverage_new.txt && mv coverage_new.txt coverage.txt
```

Comment:
I added this to remove the extra `mode: atomic` and empty lines caused by multiple `go test` invocations. It seems that codecov accepts the original form but the official `go tool cover` refuses them.

Reply:
Actually I think a `go test ./...` works here. The `vendor` directory is by default excluded from the test.

Reply:
Yeah, before go1.10 it wasn't possible to test multiple packages with coverage enabled.